### PR TITLE
Add CHANGELOG and version updates

### PR DESCRIPTION
**Pull request by nturaga**
@gpcarr @rtitle This is just a straight rebuild of the image because the parent image 'bioconductor/bioconductor_docker:RELEASE_3_12' involved critical bug fixes. 

Let me know if this checks out fine, I'm able to push the new image to the GCR.

### CODE REVIEW

File: anvil-rstudio-bioconductor/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.0.11 - 5/4/2021
+
+- Update in base image `biconductor/bioconductor_docker:RELEASE_3_12`.
```
Comment (gpcarr) [inline anvil-rstudio-bioconductor/CHANGELOG.md:3]: was there no version bump of the base image for the bug fix?

Comment (nturaga) [reply anvil-rstudio-bioconductor/CHANGELOG.md:3]: We mark bug fixes internally inside the Dockerfile, https://github.com/Bioconductor/bioconductor_docker/blob/RELEASE_3_12/Dockerfile#L10

**Comment by gpcarr** [comment]
looks good - just need to update the [readme](https://github.com/anvilproject/anvil-docker/blob/master/anvil-rstudio-bioconductor/README.md) to reflect latest version as well

**Comment by nturaga** [comment]
> looks good - just need to update the [readme](https://github.com/anvilproject/anvil-docker/blob/master/anvil-rstudio-bioconductor/README.md) to reflect latest version as well

Done!

**Review by gpcarr — APPROVED** [review]
LGTM 👍 next step would be to merge this PR then to build and push the new image with tag `us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.11` to GCR and then PR-ing a version update [here](https://github.com/DataBiosphere/terra-docker/blob/master/config/community_images.json#L14) - once that's done, I can work on getting the new image integrated with Leonardo

**Review by rtitle — APPROVED** [review]
lgtm

**Pull request merged**